examples/full_stack/variables.tf

```diff
@@ -29,18 +29,14 @@ variable "az" {
   description = "The AWS availability zone"
 
   default = {
-    eu-west-1      = "a,b,c"
```

Comment:
👌 Might bring it back later if i use the example subdir as template files but that's further down the line that we can worry about later.